app/models/organization.js

```diff
@@ -11,4 +11,5 @@ export default Owner.extend({
   iconLargeUrl: attr(),
   projects: DS.hasMany('project', { async: true }),
   members: DS.hasMany('user', { async: true }),
+  organizationMemberships: DS.hasMany('organization-membership', { async: true }),
```

Comment:
No, I mean that we need the explicit `organization-membership` everywhere, even if it might not necessarily be used in that context. Omitting a relationship in one place can cause a lot of weirdness elsewhere in the app (objects missing when you think they should be accessible).